packages/babel-preset-es2016/README.md

```diff
@@ -0,0 +1,35 @@
+# babel-preset-es2016
```

Comment:
I would post a link to the es2016 spec, a summary, etc too for more of a description. Also when we land this we should update the docs on the site http://babeljs.io/docs/plugins/#presets. Better is https://github.com/babel/babel.github.io/issues/528

http://espadrine.github.io/New-In-A-Spec/es2016/
http://www.ecma-international.org/ecma-262/7.0/index.html